scripts/ci_script.sh

```diff
@@ -17,11 +17,27 @@ if [[ $GROUP == python ]]; then
 fi
 
 
-if [[ $GROUP == js ]]; then
+if [[ $GROUP == js* ]]; then
+
+    jlpm build:packages
```

Comment:
Is this here because the windows JS tests are still run as a single job?

Reply:
Let me see if we can just build a subgroup of the packages. Not sure how well that works..

Reply:
Thanks! That cut a lot of the test runs from 5m to 3m - a huge savings!